apps/system/js/browser_context_menu.js

```diff
@@ -16,11 +18,15 @@
     this.app = app;
     this.containerElement = app.element;
     this.app.element.addEventListener('mozbrowsercontextmenu', this);
+    SettingsListener.observe(PINNING_PREF, '', function(value) {
```

Comment:
Do we need to call .unobserve() somewhere so we don't create another memory leak like bug 1193051?